### DISCUSSION TITLE
When: 8 July --- 2000 UTC+8/1400 CET/1200 GMT/0800 EDT

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1729] [General]
When: 8 July --- 2000 UTC+8/1400 CET/1200 GMT/0800 EDT

Where: [via Matrix](https://matrix.to/#/#mesa-dev:matrix.org)

Proposed Agenda:

- Anything left from [last month](https://github.com/projectmesa/mesa/discussions/1684)
- 2.0 release - what is left? Let's try to button up this weekend.

@tpike3 @rht @wang-boyu @Tortar @EwoutH Please feel free to add or takeaway from this agenda

Also, please tag any thing you think needs to go into the 2.0 release, whether it is a PR or ticket, so we can make sure we have captured everything.

**Comment by tpike3** [comment]
I will be there!
